samples/matter/thermostat/src/temperature_manager.cpp

```diff
@@ -38,16 +38,23 @@ uint8_t ReturnRemainderValue(int16_t Value)
 
 CHIP_ERROR TemperatureManager::Init()
 {
-	app::DataModel::Nullable<int16_t> temp;
+	app::DataModel::Nullable<int16_t> internalTemp;
+	app::DataModel::Nullable<int16_t> externalTemp;
 
 	PlatformMgr().LockChipStack();
-	Thermostat::Attributes::LocalTemperature::Get(kThermostatEndpoint, temp);
+	Thermostat::Attributes::LocalTemperature::Get(kThermostatEndpoint, internalTemp);
+	Thermostat::Attributes::OutdoorTemperature::Get(kThermostatEndpoint, externalTemp);
 	Thermostat::Attributes::OccupiedCoolingSetpoint::Get(kThermostatEndpoint, &mCoolingCelsiusSetPoint);
 	Thermostat::Attributes::OccupiedHeatingSetpoint::Get(kThermostatEndpoint, &mHeatingCelsiusSetPoint);
 	Thermostat::Attributes::SystemMode::Get(kThermostatEndpoint, &mThermMode);
 	PlatformMgr().UnlockChipStack();
 
-	mCurrentTempCelsius = temp.Value();
+	if(!(externalTemp.IsNull())){
```

Comment:
Could you install and configure clang-format in VS Code to have the code auto-formatted like other Matter samples? E.g. here there are some missing spaces after `if` and before `{` :).

Reply:
I have Clang formater installed and configured on autoFormat :'( But maybe something is not working properly, I will check this!